srcDoc/tutorials/performance_validation.rst

```diff
@@ -0,0 +1,642 @@
+
+The initial summary **Perftest** shows is the following:
+
+    .. code::
+
+        RTI Perftest 3.0.0 (RTI Connext DDS Micro 3.0.0)
+
+        Mode: LATENCY TEST (Ping-Pong test)
+
+        Perftest Configuration:
+            Reliability: Reliable
+            Keyed: No
+            Publisher ID: 0
+            Latency count: 1 latency sample every 1 samples
+            Data Size: 32, 64, 128, 256, 512, 1024, 2048, 4096, 8192, 16384, 32768, 63000
+            (Set the data size on the subscriber to the maximum data size to achieve best performance)
+            Publication Rate: Unlimited (Not set)
+            Execution time: 20 seconds
+            Receive using: Listeners
+            Domain: 1
+
+        Transport Configuration:
+            Kind: UDPv4
+            Nic: eth0
+            Use Multicast: False
+
+And these are the results (taken from the publisher side):
+
+Latency Results -- RTI Connext DDS Micro (UDPv4)
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+    .. csv-table::
+        :align: center
+        :header-rows: 1
+
+        "Size", "Ave (us)", "Std (us)", "Min (us)", "Max (us)", "50% (us)", "90% (us)", "99% (us)", "99.99% (us)", "99.9999% (us)"
+        32,560,158.9,361,6121,551,652,838,6070,6121
+        64,572,139.4,382,7642,567,665,861,5958,7642
+        128,609,135.6,431,5897,600,687,869,5716,5897
+        256,670,115.0,489,5394,660,749,936,5224,5394
+        512,725,130.1,551,6414,716,799,1002,5175,6414
+        1024,868,366.8,676,36814,851,938,1133,6913,36814
+        2048,1095,162.8,879,6341,1088,1177,1433,6341,6341
+        4096,1309,453.6,1083,38591,1292,1379,1643,38591,38591
+        8192,1666,167.7,1349,6790,1651,1769,2032,6790,6790
+        16384,2416,628.4,2146,39850,2396,2516,2844,39850,39850
+        32768,4046,246.8,3732,8894,4042,4161,4594,8894,8894
+        63000,6909,176.8,6564,9529,6896,7102,7368,9529,9529
+
+
+Understanding the Results
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Lets go first with the throughput results and plot all the different tests together:
+
+.. image:: performance_validation_files/Throughput_lineal.svg
+
+The first think we see is that at 5KB we are already close to saturate the
+network in all cases, which is something really good to see, but lets focus
+in the behavior for smaller samples. Lets plot the same results with a
+logarithmic scale:
+
+.. image:: performance_validation_files/Throughput_log.svg
+
+Now we can extract more information about the graphs:
+
+1. If we take out the test where we make use of *batching* we can see that using
+   Raw Transport (plain sockets) gives us the best performance.
+
+2. **Connext DDS Pro** and **Connext DDS Micro** behave in a really similar way,
+   being the latter slighly better.
+
+3. The use of *batching* really makes a difference for small samples sizes.
+
+4. After 5kB we can consider that all the tests are able to reach more than a 95%
+   of the network utilization, which all what these boards can offer.
```

Comment:
replace:

which all what these boards can offer.

with:

which is the maximum bandwidth supported by the NICs

Reply:
Done